app/i18n/en/install.php

```diff
@@ -93,6 +96,9 @@
 	'delete_articles_after' => 'Remove articles after',
 	'fix_errors_before' => 'Please fix errors before skipping to the next step.',
 	'javascript_is_better' => 'FreshRSS is more pleasant with JavaScript enabled',
+	'js' => array(
+		'confirm_reinstall' => 'You will lose your previous configuration by reinstalling FreshRSS. Are you sure you want to continue?',
```

Comment:
Maybe consider a message a bit less intimidating, such as indicating that the data (e.g. feeds/articles) will not be deleted